src/index.html

```diff
@@ -1,17 +1,246 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <fieldset class="block">
+      <legend>Personal information</legend>
+      <div>
+        <form class="form-field input"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()">
+
+          <label for="Surname"
+        class="field">Surname:
+            <input type="text"
+          name="Surname"
+          id="Surname"
+          autocomplete="off">
+          </label>
```

Comment:
```suggestion
          <label 
            for="Surname"
            class="field"
          >
            Surname:
            <input 
              type="text"
              name="Surname"
              id="Surname"
              autocomplete="off"
            >
          </label>
```
Поправь форматирование везде, где нужно